commands/cli.go

```diff
@@ -88,8 +92,23 @@ func BuildContextualMenu() []cli.Command {
 					plaintext = []byte(base64.StdEncoding.EncodeToString(plaintext))
 				}
 
-				// Init a KMS client
-				client := cloud.NewGCPKMS(c.String("key"))
+				// Work with the envelope's provided key or switch to CLI flags/env
```

Comment:
Gah, i should have probably refactored this chunk into another commit and a pr all to itself. The add method was a bit broken post key-uri-in-envelope. I didn't notice it would never attempt to open/read/use-the-key-in-envelope-format - always defaulting to ENV. Which can be problematic when the expectation is to only need to specify --key once in a great while once its in the env it should be favored.

Reply:
I think it should be fine to include in this PR